source/CardGames.py

```diff
@@ -102,7 +102,13 @@ def addCard(self, card: Card, isKnown: bool = True):
 
   def clearHand(self):
     self.hand = []
-    self.knownCards = []
+    self.knownCards = [1]
```

Comment:
I'm not sure why you edited this part of the clearHand() function. It breaks the function.